source/templates.tex

```diff
@@ -7026,30 +7072,11 @@
 \end{example}
 
 \pnum
-When an explicit template argument list is specified, if the template
-arguments are not compatible with the template parameter list or do
-not result in a valid function type as described below, type
-deduction fails.  Specifically, the following steps are performed when
-evaluating an explicitly specified template argument list with respect
-to a given function template:
-\begin{itemize}
-\item If the specified template arguments do not match the template
-parameters in
-kind (i.e., type, non-type, template), or if there
-are more arguments than there are parameters
-and no parameter is a template parameter pack, or if there is not
-an argument for each non-pack parameter,
-type deduction fails.
-
-\item If any non-type argument does not match the type of the
-corresponding non-type
-template parameter, and is not convertible to the type of the
-corresponding non-type parameter as specified in~\ref{temp.arg.nontype},
+When an explicit template argument list is specified, if the
+given \grammarterm{template-id} is invalid\iref{temp.names},
 type deduction fails.
-
-\item The specified template argument values are substituted for the
+The specified template argument values are substituted for the
```

Comment:
Add an "Otherwise" to the start of this sentence for clarity and flow.

Reply:
... and pushed as editorial commit.